docs/user-guide/styling-your-docs.md

```diff
@@ -30,12 +30,11 @@ the [custom themes](#custom-themes) section below.
 
 ![ReadTheDocs](https://docs.readthedocs.org/en/latest/_images/screen_mobile.png)
 
-## Bootstrap and Bootswatch Themes
+## Bootstrap and Bootswatch themes
```

Comment:
LGTM. 👍

At some point we need to do a review of the documentation style, but we probably need a better writer (than me) involved in the project.